src/libaktualizr/bootloader/bootloaderif.h

```diff
@@ -0,0 +1,17 @@
+#ifndef BOOTLOADERIF_H_
+#define BOOTLOADERIF_H_
+
+class BootloaderIF {
```

Comment:
Perhaps, if we just make methods virtual in Bootloader then we won't need to introduce  a new entity in libaktualizr (BootloaderIF) and introduce an interface in aklite project or just extend Bootloader. The goal is to minimize changes to libaktualizr.

Reply:
um, ok but sure I can do that. will rework.